react/src/services/foodResourcesService.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const foodResourcesService = {
+  endpoint: "http://localhost:5197/api/FoodResources",
+};
+
+foodResourcesService.getFoodResources = () => {
+  const config = {
+    method: "GET",
+    url: foodResourcesService.endpoint,
+    // withCredentials: true,
+    crossdomain: true,
+    headers: { "Content-Type": "application/json" },
+  };
+
+  return axios(config);
```

Comment:
Is it returning the correct data and returning the correct error if applicable?
  return axios(config)
    .then((response) => response.data)
    .catch((error) => {
      return Promise.reject(error);
    });
};